source/_components/bmw_connected_drive.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "BMW connected drive"
+description: "Instructions on how to setup your BMW connected drive account with Home Assistant."
+date: 2018-01-10 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.65
+---
+
+This component lets you retrieve data on your BMW vehicle from the BMW Connected Drive portal. You need to have a working BMW Connected Drive account and a Connected Drive enabled vehicle for this to work.
+
+For compatibility with your BMW vehicle check the (bimmer_connected page)[https://github.com/ChristianKuehnel/bimmer_connected] on github.
+
+To enable this component in your installation, add the following to your
+`configuration.yaml` file:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+bmw_connected_drive:
+  mycar:
+    username: #your user name
+    password: #your_password>
+    country: #country of your Connected Drive account
+```
+{% endraw %}
+
+
+{% configuration %}
+bmw_connected_drive:
+    description: configuration
+    required: true
+    type: map
+    keys:
```

Comment:
This config layout is more complex than needed. You only need to add the keys under the component, not the component itself.

Reply:
fixed